.sandstorm/sandstorm-pkgdef.capnp

```diff
@@ -24,10 +24,10 @@ const pkgdef :Spk.PackageDefinition = (
       categories = [developerTools,],
       author = (
         upstreamAuthor = "Christian Gierke and Kay Sievers",
-        contactEmail = "david@sandstorm.io",
-        pgpSignature = embed "pgp-signature",
+        contactEmail = "abliss@gmail.com",
+        pgpSignature = embed "pgp-signature-abliss",
       ),
-      pgpKeyring = embed "keyring",
+      pgpKeyring = embed "keyring",p
```

Comment:
stray `p`?

Reply:
sorry, I had pushed prematurely. now updated with the correct changes, and I put the new spk at https://github.com/abliss/gitwebpages-sandstorm/releases/tag/0.0.10